openshift/dataverse-broker.yaml

```diff
@@ -10,6 +10,14 @@ objects:
     name: dataverse-broker
   spec:
     url: http://dataverse-broker.bdc.svc.cluster.local
+    insecureSkipTLSVerify: false
+    caBundle: ${CA_BUNDLE}
+    authInfo: 
+      bearer: 
+        secretRef:
+          kind:      Secret
+          name:      dataversebroker-client
```

Comment:
In order to have a static secret name with the service account token, you'll need to emulate this example: https://github.com/openshift/origin/blob/master/install/templateservicebroker/apiserver-template.yaml#L116

Reply:
You should refer to the service account token for `dataverse-broker` service account into the `dataversebroker-client` secret if you don't want to make other changes here.